docs/source/tutorial/00-introduction.mdx

```diff
@@ -27,10 +26,9 @@ The tutorial uses the following tools and frameworks:
 * [ViewBinding](https://developer.android.com/topic/libraries/view-binding), [Navigation Component](https://developer.android.com/guide/navigation/navigation-getting-started), [ConstraintLayout](https://developer.android.com/training/constraint-layout) and [RecyclerView](https://developer.android.com/guide/topics/ui/layout/recyclerview).
 * [Coil](https://github.com/coil-kt/coil) for image loading.
 
-To focus its scope, this tutorial uses a starter project available at https://github.com/apollographql/apollo-android-3-tutorial so you can
-focus on the GraphQL parts.
+To focus its scope, this tutorial uses a starter project available at [https://github.com/apollographql/apollo-android-3-tutorial](https://github.com/apollographql/apollo-android-3-tutorial) so you can focus on the GraphQL parts.
```

Comment:
I'm not sure "to focus its scope" is a correct expression?  Alternative proposal below. Feel free to dismiss if it was correct.

```suggestion
To focus on the important parts, this tutorial uses a starter project available at [https://github.com/apollographql/apollo-android-3-tutorial](https://github.com/apollographql/apollo-android-3-tutorial) so you don't have to deal with project setup and boilerplate.
```